### PR TITLE
[ticket/16080] ACP - Provide meaningful error msg for deleted installed style

### PR DESCRIPTION
**Pull request by 3D-I**
That's a very edge case.

PHPBB3-16080

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16080

### CODE REVIEW

**Comment by 3D-I** [comment]
!set php

**Comment by 3D-I** [comment]
I opted for a meaningful error message instead.

immagine

**Comment by 3D-I** [comment]
!set Language/Text

File: phpBB/includes/acp/acp_styles.php [hunk 1]
```diff
@@ -1123,7 +1123,13 @@ public function sort_styles($style1, $style2)
 	*/
 	protected function read_style_cfg($dir)
 	{
+		if (!file_exists($this->styles_path . $dir . '/style.cfg'))
+		{
+			trigger_error($this->user->lang['NO_STYLE_CFG'] . $dir, E_USER_WARNING);
```
Comment (VSEphpbb) [inline phpBB/includes/acp/acp_styles.php]: preferred way would be:
`$this->user->lang('NO_STYLE_CFG', $dir)`
and 
`'Could not get the style configuration file for: %s'`

Comment (3D-I) [reply phpBB/includes/acp/acp_styles.php]: Yup, thx. Let me test it first.

File: phpBB/includes/acp/acp_styles.php [hunk 2]
```diff
@@ -1123,7 +1123,14 @@ public function sort_styles($style1, $style2)
 	*/
 	protected function read_style_cfg($dir)
 	{
+		// This should never happen, we give them a red warning because of its relevance.
+		if (!file_exists($this->styles_path . $dir . '/style.cfg'))
+		{
+			trigger_error(sprintf($this->user->lang['NO_STYLE_CFG'], $dir), E_USER_WARNING);
```
Comment (marc1706) [inline phpBB/includes/acp/acp_styles.php]:
```suggestion
			trigger_error($this->user->lang('NO_STYLE_CFG', $dir), E_USER_WARNING);
```

Comment (3D-I) [reply phpBB/includes/acp/acp_styles.php]: I will try your suggestion @marc1706 - Again I mean :)

Comment (3D-I) [reply phpBB/includes/acp/acp_styles.php]: Ok, dunno why didn't work yesterday.
